docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,32 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
+
+Before reading the following, also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Does your content really need images?
+Generally, we would avoid adding new images to our content.
+
+In patterns and components, we should be using code examples when we want to convey what our pattern or component will look like. Avoid using screenshots of patterns or components wherever possible.
+
+## Can your image be an SVG?
+SVG's are usually better for web performance as they scale with different screen sizes and load faster than most image formats of the same size. If your image is a "graphic" (in other words it isn't a photo or a screenshot) it should be converted to an SVG and added to the page directly.
+
+This doesn't however mean that SVG's can't also have a negative impact on performance if their file size is unnecessarily large. Consider using a compression service such as [SVGOMG](https://jakearchibald.github.io/svgomg/) to ensure your SVG is as performant as possible.
+
+## Ensure your image is an appropriate file size
+If your image can't or shouldn't be an SVG is instead a "standard" file type eg: PNG, JPEG etc, avoid adding very large file size images to the website.
```

Comment:
Thanks for this write up @owenatgov

I'll give it a check over for technical bits, will let @claireashworth take a look too

Probably want to squash a few bits that fall under [**Contractions, abbreviations and acronyms**](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#contractions-abbreviations-and-acronyms)?

>Avoid:
>
>* should’ve, could’ve, would’ve, they’ve - these can be hard to read
>* negative contractions like ‘can’t’ and ‘don’t’ - some users find them harder to read, or misread them as the opposite of what they say